### ISSUE TITLE
Docs: Issue @ tsconfig#moduleResolution

### ISSUE THREAD

**Issue by AjayChambers** [microsoft/TypeScript-Website#2224]
**Page URL:** [tsconfig#moduleResolution](https://www.typescriptlang.org/tsconfig#moduleResolution)

**Issue:** I would have fixed this myself, but sometimes I can make mistakes, and makes me nervous changing a doc under the circumstance, that there is an error in the docs. If I contributed more to this project I would probably feel more comfortable making the change. With that said, there is an issue where you have to use the nightly build to get support for the  moduleResolution property in `tsconfig.json`. 

#### [This is the issue](https://github.com/microsoft/TypeScript/issues/47311)

---

In a nutshell `nodenext`, and `node12`, both will not work in version `v4.5.4`, however the documentation says it will work w/ version `v4.5.4` onward. The way it is now, in-order to get support for node12 module resolution, you have to be using the `unstable v4.6.x nightly build`.

Anyways, hopefully this is useful.

**Recommended Fix:**  If I am correct, you can probably just change it from v4.5.4 onward, to currently need to be using the nightly build.
